src/components/Footer.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import {MdCopyright} from 'react-icons/md'
 
 const Footer = () => {
     return (
         <div>
-            <p>Made by chmette and olhanotolga, based on weibenfalk’s tutorial</p>
+            <p> <MdCopyright/>Made by  <a href="https://github.com/chmette">chmette</a> and <a href="https://github.com/olhanotolga">olhanotolga</a>, based on <a href="https://www.youtube.com/watch?v=ZGOaCxX8HIU">weibenfalk’s tutorial</a> </p>
```

Comment:
Let's not use a copyright symbol here, we don't own the code, after all :)

Reply:
i cant see , where you mean . Do you want to set it after 'based on '?